src/NATS.Client.JetStream/Models/ConsumerCreateAction.cs

```diff
@@ -0,0 +1,8 @@
+namespace NATS.Client.JetStream.Models;
+
+public enum ConsumerCreateAction
+{
+    Default = 0,
```

Comment:
Would it make sense to make the default descriptive?  I.e. `CreateOrUpdate = 0` instead of `Default = 0`?

Or is it best to leave it vague as the NATS Server can decide the default?